packages/@webex/plugin-meetings/test/unit/spec/breakouts/breakout.ts

```diff
@@ -105,6 +105,7 @@ describe('plugin-meetings', () => {
 
         onBreakoutMoveRequestStub.restore();
         onBreakoutMoveResponseStub.restore();
+        submitClientEventStub.restore()
```

Comment:
What happens if we don't have this restore?

Reply:
Same as above. Restore is required.